docs/cmdline-opts/form-escape.d

```diff
@@ -0,0 +1,10 @@
+Long: form-escape
+Help: Escape multipart form field/file names using backslash
+Protocols: HTTP
+See-also: form
+Added: 7.80.0
+Category: http post
+Example: --form-escape --form 'field\\name=curl' 'file=@load"this' $URL
+---
+Tells curl to escape multipart form field and file names using the
+backslash-escaping algorithm rather than the percent-encoding.
```

Comment:
How about:

Tells curl to pass on multipart form fields and file names using backslash-escaping instead of percent-encoding.

Reply:
It seems to me that having a plural for `fields` references the contents rather than their names. What about `fields- and file names `?

Reply:
Changed it to:

Tells curl to pass on names of multipart form fields and files using backslash-escaping instead of percent-encoding.